workers/refresh_config_data_sources_worker.rb

```diff
@@ -8,8 +9,8 @@ def work
       FastlaneCI::Services.project_service.refresh_repo
     end
 
-    def sleep_interval
-      15
+    def scheduler
+      WorkerScheduler.new(sleep_interval: 15)
```

Comment:
I have these being declared on init in another CL

Reply:
❤️